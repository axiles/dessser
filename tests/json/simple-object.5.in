  {

		"bar"  :{"glop":true}
,"foo"
:9E-16}
