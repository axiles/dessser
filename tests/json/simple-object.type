{
  foo: float;
  bar: {
    baz: i32?[]?;
    glop: bool;
  }?;
  zap: i8[2][[]]?;  // list of vectors
}?
