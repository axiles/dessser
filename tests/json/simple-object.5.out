(0x1.036847569cd7ap-50 (null T) null)
